### DISCUSSION TITLE
Fixed width font in comment textarea

### DISCUSSION THREAD

**Discussion by michaldudak** [community/community#5442] [Other Feature Feedback, Questions, & Ideas] [answered]
Since comments in issues and pull requests often contain code examples and markdown, it'll be more convenient if we were able to author them with a monospace font. At least an option to use fixed-width font in the editor would be enough.

I am aware that this can be achieved with browser add-ons, but having such functionality built-in would be great.

**Comment by willsmythe** [comment, marked as answer]
Now available! https://twitter.com/GHchangelog/status/1448033685723353092

**Reply by michaldudak** [reply]
Perfect! Thanks!

**Comment by asherber** [comment]
I like this feature, but it would be even more useful if it picked up some of the functionality from the [Monaco Editor for GitHub](https://chrome.google.com/webstore/detail/monaco-markdown-editor-fo/mmpbdjdnmhgkpligeniippcgfmkgkpnf) extension.

In particular, I like the fact that the extension provides basic Markdown and syntax highlighting and plays a little nicer with indents in fenced code blocks. The extension also allows for changing the font and font size, among many other tweaks.

(The main drawbacks to the extension itself are that it doesn't support things like @ and # in text boxes, and it removes the text box resize grab handle in Chrome.)
